File: Proyecto/Windows phone/Messages/Messages/Model/MessageFrom.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Messages.Model
{
    public class MessageFrom
    {
        public int id { get; set; }
        public int from { get; set; }
        public int to { get; set; }
        public String text { get; set; }
        public String date { get; set; }
        public MessageFrom()
        {

        }
    }
}

File: Proyecto/iOS/Messages/Messages/MessageFrom.swift
//
//  MessageFrom.swift
//  Messages
//
//  Created by Estudiantes on 3/06/16.
//  Copyright © 2016 Estudiantes. All rights reserved.
//

import Foundation
import JSONHelper
struct MessageFrom:Deserializable {
    var id=0
    var from=0
    var to=0
    var text=""
    var date=""
    init(data: [String: AnyObject]) {
        id <-- data["id"]
        from <-- data["from"]
        to <-- data["to"]
        text <-- data["text"]
        date <-- data["date"]
    }
}